jwst/ami/lg_model.py

```diff
@@ -394,23 +378,14 @@ def fit_image(
         self.fringeamp, self.fringephase = leastsqnrm.tan2visibilities(self.soln)
         self.fringepistons = utils.fringes2pistons(self.fringephase, len(self.ctrs))
         self.redundant_cps = leastsqnrm.redundant_cps(self.fringephase, n=self.N)
-        self.t3_amplitudes = leastsqnrm.t3_amplitudes(self.fringeamp, n=self.N) # RC 8/24
+        self.t3_amplitudes = leastsqnrm.t3_amplitudes(
```

Comment:
I think this would be better putting the `#RC 8/24` before this computation and put the computation back on one line.

Reply:
Nice catch, it seems the formatter didn't necessarily find the best way to split it over multiple lines here